src/test/java/com/ge/predix/uaa/token/lib/ZacTokenServiceTest.java

```diff
@@ -57,45 +57,62 @@
     public void testLoadAuthentication() {
         // testing when zone id is not null
         String zoneUserScope = SERVICEID + ".zones." + ZONE + ".user";
-        assertAuthentication(loadAuthentication(PREDIX_ZONE_HEADER_NAME, BASE_DOMAIN, ZONE, zoneUserScope), ZONE);
+        assertAuthentication(loadAuthentication(PREDIX_ZONE_HEADER_NAME, BASE_DOMAIN, ZONE, zoneUserScope, true), ZONE);
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
Not reviewing this class for now.    Wasnt expecting so many changes to this class for this change.